conans/model/editable_cpp_info.py

```diff
@@ -35,23 +33,25 @@ def _loads(cls, content, require_namespace):
         """ Returns a dictionary containing information about paths for a CppInfo object: includes,
         libraries, resources, binaries,... """
 
-        parser = configparser.ConfigParser(allow_no_value=True, delimiters=('#', ))
+        parser = configparser.ConfigParser(allow_no_value=True)
         parser.optionxform = str
-        parser.read_string(content)
+        content_file_obj = StringIO(content)
+        parser.readfp(content_file_obj)  # FIXME: Deprecated in Python 3
```

Comment:
What about moving the parsing effort to the functions `load` (`parser.read(filepath)`) and `loads` (workaround with the `readfp`, or try/catch depending on python version) and keeping code here clean?